docs/src/modules/operations/pages/index.adoc

```diff
@@ -1,9 +1,27 @@
-= Operating
+= Fully-automated regions
```

Comment:
I think this could be `Fully-managed Operations` ... vs `Self-managed Operations` (below).. makes the contrast easier to understand in my view.